domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/pathways/Pathway.java

```diff
@@ -0,0 +1,400 @@
+         */
+        public PathwayBuilder traversalTime(@Nullable final Integer traversalTime) {
+            this.traversalTime = traversalTime;
+            return this;
+        }
+
+        /**
+         * Sets field stairCount value and returns this
+         *
+         * @param stairCount number of stairs of the pathway
+         * @return builder for future object creation
+         */
+        public PathwayBuilder stairCount(@Nullable final Integer stairCount) {
+            this.stairCount = stairCount;
+            return this;
+        }
+
+        /**
+         * Sets field maxSlope value and returns this
+         *
+         * @param maxSlope maximum slope ratio of the pathway
+         * @return builder for future object creation
+         */
+        public PathwayBuilder maxSlope(@Nullable final Float maxSlope) {
+            this.maxSlope = maxSlope;
+            return this;
+        }
+
+        /**
+         * Sets field minWidth value and returns this
+         *
+         * @param minWidth  minimum width of the pathway in meters
+         * @return builder for future object creation
+         */
+        public PathwayBuilder minWidth(@Nullable final Float minWidth) {
+            this.minWidth = minWidth;
+            return this;
+        }
+
+        /**
+         * Sets field signpostedAs value and returns this
+         *
+         * @param signpostedAs string of text from physical signage visible to transit riders
+         * @return builder for future object creation
+         */
+        public PathwayBuilder signpostedAs(@Nullable final String signpostedAs) {
+            this.signpostedAs = signpostedAs;
+            return this;
+        }
+
+        /**
+         * Sets field reversedSignpostedAs value and returns this
+         *
+         * @param reversedSignpostedAs same than the @param signPostedAs field, but when the pathways is used backward
+         * @return builder for future object creation
+         */
+        public PathwayBuilder reversedSignpostedAs(@Nullable final String reversedSignpostedAs) {
+            this.reversedSignpostedAs = reversedSignpostedAs;
+            return this;
+        }
+
+        /**
+         * Return a {@code EntityBuildResult} representing a row from pathways.txt if the requirements from the official
+         * GTFS specification are met. Otherwise, method returns a collection of notices specifying the issues.
+         *
+         * @return an {@code EntityBuildResult} representing a row from pathways.txt if the requirements from the
+         * official GTFS specification are met. Otherwise, method returns a collection of notices specifying the issues.
+         */
+        public EntityBuildResult<?> build() {
+            noticeCollection.clear();
+
+            if (pathwayId == null ||
+                    fromStopId == null ||
+                    toStopId == null ||
+                    !IsBidirectional.isEnumValueValid(originalIsBiDirectionalInteger) ||
```

Comment:
I think this situation is handled:

if `originalIsBiDirectionalInteger` is null, method `fromInt` of enum `IsBidirectional` returns `null`. 

This being done, we then check if `originalIsBiDirectionalInteger=null` is valid with method `isEnumValueValid` which will return `false`.

 After that, since `isBidirectional` and `originalIsBiDirectionalInteger` are null we emit a new `MissingRequiredValueNotice`

Reply:
Also, I think that this test covers this situation: https://github.com/MobilityData/gtfs-validator/blob/b8e25a006d8c5ddd56e3733be34649aedaddd08f/domain/src/test/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/pathways/PathwayTest.java#L250